src/main/java/meteordevelopment/meteorbot/MeteorBot.java

```diff
@@ -66,6 +70,7 @@ public void onReady(@Nonnull ReadyEvent event) {
 
         DONATOR_CHAT = GUILD.getGuildChannelById(862376330695016488L);
         SUPPORT = GUILD.getTextChannelById(862489308497313822L);
+        LOGS_CHANNEL = GUILD.getTextChannelById(861351437997178953L);
```

Comment:
Its currently set to jenral in the test server because there werent any logs channel